### ISSUE TITLE
[FEATURE REQUEST] subtasks to TODO list

### ISSUE THREAD

**Issue by D3faIt** [lfos/calcurse#261]
Hi.

I've been looking for an application which supports task management.
I wish there was an option to add subtasks to TODO items.

Something like this:
![Screenshot_20200211_163238](https://user-images.githubusercontent.com/8147434/74251259-27af1580-4cec-11ea-8bec-c693be9d6bbd.png)

Is this a kind of feature that would be *possible* to implement?

**Comment by betaAlbireo** [comment]
Yes, that would be really great! I'm also looking for this. Have you found a solution or alternative application yet?

**Comment by lfos** [comment]
It would definitely be possible. If you're fine with the subtasks not being displayed in the main panel, you can already achieve this by adding notes to the items.

**Comment by y10011010** [comment]
@lfos are conflicts across [multiple calendars](#346) relevant, perhaps?
